test/e2e/operator.go

```diff
@@ -436,18 +436,20 @@ var _ = Describe("ARO Operator - ImageConfig Reconciler", func() {
 
 	// Reimplementation of function from image config controller
 	getCloudAwareRegistries := func(instance *arov1alpha1.Cluster) ([]string, error) {
-		var requiredRegistries []string
+		acrSubdomain := strings.Split(instance.Spec.ACRDomain, ".")[0]
```

Comment:
I am not happy about this re-implementation, but this is work for a separate task.

Reply:
Agreed.